lang/cs/Org.Apache.REEF.Client/Common/HttpClientHelper.cs

```diff
@@ -90,7 +91,12 @@ internal static List<string> GetRmUri(string filePath)
             }
         }
 
-        internal static string GetAppId(string filePath)
+        /// <summary>
+        /// Return teh applicaiton id of the Job
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        public string GetAppId(string filePath)
```

Comment:
Does this have to be `public`? It seems like a hack, ahm, implementation detail we won't want to support long term.